### ISSUE TITLE
Limit thread title length, point to body field

### ISSUE THREAD

**Issue by mxstbr** [withspectrum/spectrum#4832]
https://alpha.spectrum.chat/spectrum/general/very-loooong-titles-and-no-body~fbde6827-0d27-475e-8e5e-1fa673f71bfc

**Comment by brianlovin** [comment]
Agreed we should probably cap this at some length...140 chars? Not sure. We do truncate it everywhere else in the UI anyways.
